.github/actions/pulumi_ephemeral_deploy/action.yml

```
@@ -68,12 +73,29 @@ runs:
-      uses: aws-actions/configure-aws-credentials@v5.1.0
+      uses: aws-actions/configure-aws-credentials@v5.1.1
       with:
         role-to-assume: arn:aws:iam::${{ inputs.aws-account-id }}:role/${{ inputs.aws-role-name }}
         aws-region: ${{ inputs.aws-region }}
 
     - name: Run CLI
-      working-directory: ${{ github.workspace }}/${{ inputs.project-dir }}
-      run: uv run python -m ${{ inputs.deploy-script-module-name }}.${{ inputs.deploy-script-name }} --stack=${{ inputs.stack-name }} ${{ inputs.cli-action }}
-      shell: bash
+      # apparently getting the stdout is nigh impossible in GHA, so this action helps
+      uses: mathiasvr/command-output@34408ea3d0528273faff3d9e201761ae96106cd0 # ratchet:mathiasvr/command-output@v2.0.0
```

YAMLlint (1.38.0)

[warning] too few spaces before comment: expected 2

(comments)